added home page, category, featured

Hnh013 opened the pull request:
Pages added :
Home
Components added :
Navigation bar , Footer , Category , Featured Product

Back-end changes/modifications :
 - Added one route to fetch all users in database
 - Added dummy data for categories and products

lomesshh reviewed (approved):
clean code 
nice folder structure
suggestion : 
avoid using !important in css

OmKakatkar reviewed (approved):
Folder Structure looks clean
Like the way you have seperated pages and the components
Try to avoid `!important` in CSS

Hnh013 commented:
> clean code nice folder structure suggestion : avoid using !important in CSS rest all looks good to me

 hi Lomesh,  thanks for the suggestion, I have modified logic and removed !important from CSS attributes

Hnh013 commented:
> Folder Structure looks clean Like the way you have seperated pages and the components Try to avoid `!important` in CSS Rest of it looks good

Hi Om , thanks for the review , I have removed !important from CSS attributes now
